_faq/why-mixer.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Why does Istio need Mixer?
+order: 135
+type: markdown
+---
+{% include home.html %}
+
+Mixer provides a rich intermediation layer between the Istio components as well as Istio-based services,
+and the infrastructure backends used to perform access control checks and telemetry capture. This
+layer enables operators to have rich insights and control over service behavior without requiring
+changes to service binaries.
+
+Mixer is designed as a stand-alone component, distinct from Envoy, in order to be able
```

Comment:
We should be explicit about "Why is mixer not implemented as an envoy filter" 

* Proxy stability 
  * Complex logic moved to the mixer. Simpler and more reliable proxy.
  * If mixer goes down, proxy can fail open
* Proxy replaceability
  * Proxy can be swapped by using mixer client
  * Can use multiple types of proxies; nginx, envoy all in the same deployment.
* Mixer can run in a separate trust domain
 * Isolates back ends from the proxy
* Services can call the control plane directly. Solves N x M integration problem.
* Proxies are simpler and smaller
 * They can be numerous.
 * Cheaper to run
* Proxy and mixer can scale at different rates